src/connection.rs

```diff
@@ -180,7 +180,6 @@ pub enum ConnectionState {
 impl From<i64> for ConnectionState {
     fn from(state: i64) -> Self {
         match state {
-            0 => ConnectionState::Unknown,
```

Comment:
Why did you remove this? It' a valid state https://developer.gnome.org/NetworkManager/stable/nm-dbus-types.html#NMState